utils/autotailor

```diff
@@ -296,32 +257,99 @@ class Tailoring:
             for rule_id, props in tailoring["rules"].items():
                 if "evaluate" in props:
                     if props["evaluate"]:
-                        self.rules_to_select.append(rule_id)
+                        self.rules_to_select.add(rule_id)
                     else:
-                        self.rules_to_unselect.append(rule_id)
+                        self.rules_to_unselect.add(rule_id)
                 for attr in ATTRIBUTES:
                     if attr in props:
                         self.change_rule_attribute(rule_id, attr, props[attr])
 
-    def import_json_tailoring(self, json_tailoring):
-        with open(json_tailoring, "r") as jf:
-            all_tailorings = json.load(jf)
+    def to_xml(self, root):
+        profile = ET.SubElement(root, "{%s}Profile" % NS)
+        profile.set("id", self._full_profile_id(self.profile_id))
+        profile.set("extends", self._full_profile_id(self.extends))
 
-        if 'profiles' in all_tailorings and all_tailorings['profiles']:
-            if len(all_tailorings['profiles']) > 1:
-                raise ValueError("The autotailor tool currently does not support multi-profile JSON tailoring.")
-            tailoring = all_tailorings['profiles'][0]
+        # Title has to be there due to the schema definition.
+        title = ET.SubElement(profile, "{%s}title" % NS)
+        if self.profile_title:
+            title.set("override", "true")
         else:
-            raise ValueError("JSON Tailoring does not define any profiles.")
+            title.set("override", "false")
+        title.text = self.profile_title
+
+        self._add_group_select_operations(profile)
+        self._add_rule_select_operations(profile)
+        self._add_value_overrides(profile)
```

Comment:
these methods should be renamed to make it clear that they convert something to XML

Reply:
Done.